docs/psf_spec/high_level_overview.rst

```diff
@@ -0,0 +1,244 @@
+`~photutils.psf.DAOGroup`, allow for the merging of astrometrically overlapping sources
+into a set of multiple stars. These stars must be fit simultaneously (see Fitter_) as a
+composite model, avoiding the potential issue where blended sources return incorrect fluxes,
+as affects aperture photometry. The block accepts the `~astropy.table.Table` output from
+Finder_ and returns a second `~astropy.table.Table` containing the same columns as the input
+as well as an additional column indicating the group number of each source. More information
+on this block can be found in :doc:`Group Maker <group_maker>`.
+
+
+.. _Fitter:
+
+Fitter
+^^^^^^
+Once a set of sources has been detected and grouped, each source must have its respective
+properties -- position and flux primarily -- determined. For this a ``Fitter`` instance is
+required; these are drawn from a separate class or instance of minimization routines, such as
+those given under the `~astropy.modeling` umbrella. The fitter must be a two-dimensional fitter,
+by nature of the image to be fit, accepted as an input to the instance. The class must then
+return the position of the source (or sources, if a composite model was supplied; see
+`Group Maker`_) and its total flux, as determined by the PSF model describing the telescope
+system -- optics, CCD, etc. -- that created the image. See `below <PSF Model_>`_ for more
+details on the PSF model, and the :doc:`fitter <fitter>` documentation for further
+information on the ``Fitter`` instance and its properties.
+
+
+.. _PSF Model:
+
+PSF Model
+^^^^^^^^^
+Perhaps the most important aspect of PSF fitting, the ``PSF Model`` describes the distribution
+of light falling on a given pixel from a source at some position with some total flux, fully
+describing the effects of telescope optics, quantized CCD pixels, etc. In this respect it is
+more properly described as a PRF than a PSF -- see the :doc:`documentation <psf_model>` for
+more details -- but we conflate the two terms here freely. Similar to the ``Fitter``, it is
+a separate callable class detailing a specific PSF response. This can be some analytical
+function, such as `~photutils.psf.IntegratedGaussianPRF`, or an empirical description of the
+PSF, such as that implemented by the effective PSF functionality (see :ref:`build-epsf` for
+details). The model must describe a two-dimensional distribution, and must return the total
+flux of the source and its sub-pixel position via the ``evaluate`` function, accepted by the
+Fitter_ when minimizing the fit between the model and the data.
+
+
+.. _noise:
+
+Noise Description
+^^^^^^^^^^^^^^^^^
+The ``noise`` block within the PSF fitting process is, in much the same way as the ``Fitter``
+block, a separate ``callable`` function mapping the relationship between the pixel values in
+the input ``image`` and the corresponding uncertainty -- in units of standard deviations -- of
```

Comment:
I'm not sure I like the "in units of standard deviation part", but will discuss more in #755

Reply:
I've removed the subclause for now as it's not particularly necessary (this being a brief, high level overview and all) but we can discuss that further in the other PR and I'll make any changes finalised there in this branch too.